src/test/java/de/poiu/apron/io/PropertyFileReaderTest.java

```diff
@@ -425,6 +426,67 @@ public void testSingleBackslashOnLine() throws IOException {
     assertThat(readEntries.get(1)).isEqualTo(new PropertyEntry("   ", "\\ ", "   ", "", "\n"));
   }
 
+  /**
+   * This test verifies bug #11: https://github.com/hupfdule/apron/issues/11
+   * @throws IOException
+   */
+  @Test
+  public void test_CrLfEndings() throws IOException {
+    // - preparation
+    final File propertyFile= this.createTestFile(""
+        + "keyA1=valueA1\n"
+        + " keyA2  =  valueA2\n"
+        + "\tkeyA3\t=\tvalue A3\t\n"
+        + "keyA4 = very long\\\r\n"
+        + "value A4 over \\\r\n"
+        + "multiple lines\r\n"
+        + "        \n"
+        + "keyB1:valueB1\n"
+        + " keyB2 : valueB2\n"
+        + "\t keyB3\t:\t value B3 \n"
+        + "keyB4 : very long\\\n"
+        + "value B4 over \\\n"
+        + "multiple lines\\\n"
+        + "\n"
+        + "keyC1 valueC1\n"
+        + "  keyC2   valueC2\n"
+        + "\t keyC3\t\tvalue C3 \n"
+        + "keyC4   very long\\\n"
+        + "value C4 over \\\n"
+        + "\t \tmultiple lines");
+
+    // - execution
+    final List<String> readEntries= new ArrayList<>();
+    try (final PropertyFileReader reader= new PropertyFileReader(propertyFile);) {
+      Entry entry;
+      while ((entry= reader.readEntry()) != null) {
+        readEntries.add(entry.toCharSequence().toString());
+      }
+    }
+
+    // - validation
+    assertThat(readEntries.size()).isEqualTo(13);
```

Comment:
As we already checked the size of the `java.util.Properties` and want to have the same length this should be changed to:
```suggestion
    assertThat(readEntries.size()).isEqualTo(javaUtilProperties.size());
```

(which is consistent with all the other tests in this class).